README.md

```diff
@@ -3,11 +3,11 @@
 [[:books: **Additional Documents**]](#additional-documents)
 [[:arrow_forward: **Demo**]](https://ktock.github.io/container2wasm-demo/)
 
-# container2wasm: Container to WASM converter
+# container2wasm: Docker Container to WASM converter
 
 container2wasm is a container-to-wasm image converter that enables to run the container on WASM.
 
-- Converts a container to WASM with emulation by Bochs (for x86_64 containers) and TinyEMU (for riscv64 containers).
+- Converts a Docker container to WASM with emulation by Bochs (for x86_64 containers) and TinyEMU (for riscv64 containers).
```

Comment:
@ShalokShalom Thanks for your patch! But actually, container2wasm is designed to accept not only Docker image but also [OCI images](https://github.com/opencontainers/image-spec/) as well (please also refer to [the unpacking logic](https://github.com/ktock/container2wasm/blob/d25910899f2a2259d3ce7ce5b0d227877c415732/cmd/create-spec/main.go#L70) if you're interested in) so we can't limit the type of image to Docker here.

Reply:
@ShalokShalom 

> that includes Docker.

AFAIK, it's not true. OCI and Docker are two different image formats with different mediatypes (you can find the detailed compatibility matrix at the spec https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/media-types.md#compatibility-matrix). And container2wasm tries to support both of them because both of them are used in the wild.

Maybe you can change this to `Converts a container image (Docker/OCI) to WASM...`. And no change should be applied to `# container2wasm: Container to WASM converter` part.

Reply:
And please squash the commit and add the `Signed-off-by` message https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s .

Reply:
Sorry, I feel like that gets a bit too complicated